### PR TITLE
MOB - 1775 - Connect home screen to Settings and QRScanner

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift [hunk 1]
```diff
@@ -47,11 +50,58 @@ extension WalletEntity {
             let domains = createMockDomains()
             let numOfNFTs = Int(arc4random_uniform(10) + 1)
             let nfts = (0...numOfNFTs).map { _ in  NFTDisplayInfo.mock() }
+            
+            
+            let balance: [WalletTokenPortfolio] = [.init(address: $0.address,
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift:55]: Ideally mock data should be moved to some special files with 'Mock' in the name. This looks very confusing

File: unstoppable-ios-app/domains-manager-ios/Services/WalletConnect/WalletConnectService_V2/WCRequestUIConfiguration.swift [hunk 2]
```diff
@@ -22,3 +22,15 @@ enum WCRequestUIConfiguration {
     }
 }
 
+// MARK: - Open methods
+extension WCRequestUIConfiguration {
+    static func mockSign() -> WCRequestUIConfiguration {
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Services/WalletConnect/WalletConnectService_V2/WCRequestUIConfiguration.swift:27]: Also move to a special file 'Mock....'

File: unstoppable-ios-app/domains-manager-ios/SwiftUI/ViewModifiers/ViewPullUp/ViewPullUpCustomContentConfiguration.swift [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+            connectionConfiguration = connectionConfig
+        }
+        
+        signTransactionView.setRequireSA(connectionConfig.isSARequired)
+        signTransactionView.pullUp = pullUp
+        let chainIds = connectionConfiguration.appInfo.getChainIds().map({ String($0) }).joined(separator: ",")
+        let analyticParameters: Analytics.EventParameters = [.wcAppName: connectionConfiguration.appInfo.getDappName(),
+                                                             .hostURL: connectionConfiguration.appInfo.getDappHostName(),
+                                                             .chainId: chainIds]
+        //        try await withSafeCheckedThrowingMainActorContinuation(critical: false) { completion in
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/SwiftUI/ViewModifiers/ViewPullUp/ViewPullUpCustomContentConfiguration.swift:77]: Dead code?

**Review by rommex — APPROVED** [review]
With minor suggestions

File: unstoppable-ios-app/domains-manager-ios/Services/WalletConnect/WalletConnectService_V2/WCRequestUIConfiguration.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Services/WalletConnect/WalletConnectService_V2/WCRequestUIConfiguration.swift:27]: Removed

File: unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift (hunk 1 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift:55]: Moved to mock fabric

**Pull request merged**